saraho1123/currency-exchange
# Feature/limit input fields

## Is this a fix or a feature?
feature

## What is the change?
+ Adds dropdowns of all available currency codes for this api
+ A dropdown is used for both userCurrency and newCurrency

## What does it fix?
+ Makes it a lot easier to do sad path testing! :) 

## Where should the reviewer start?
+ line 3 (and 28) of Form.js
+ line 41 of sampleApiData.js

## How should this be tested?
+ clone down
+ cd into project
+ run `npm install` in terminal
+ run `npm start` in terminal
  - Form should appear. It can be filled out use the dropdowns and number field. There is a message to the user to fill out all fields. When this is completed, the 'Get Currency Conversion' appears.

## Relevant Tickets
+ #3 
+ #4 
+ #14 
+ #15
